cache.py

```diff
@@ -20,9 +21,13 @@ def add(self, url:str):
                         progressive=True,
                     ).order_by("resolution").desc().first()
         video_file_name = video.default_filename
+        UUID = uuid.uuid4()
+        theuuid = str(UUID)
         if (video_file_name not in os.listdir(self.file_path)):
             video.download(self.file_path)
         video_id = self.get_video_id(url)
+        os.rename(video_file_name, theuuid + ".mp4")
+        video_file_name = theuuid + ".mp4"
```

Comment:
```suggestion
        video_file_name = str(uuid.uuid4()) + ".mp4"
        os.rename(video_file_name, video_file_name)
```

Reply:
![image](https://github.com/SCE-Development/sce-tv/assets/113883042/29b6540b-b321-4d62-856c-414614febcc8)

gives this error when the changes are applied